hedera-node/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -214,6 +231,32 @@ public Pair<ResponseCodeEnum, Long> create(
         return Pair.of(OK, fee);
     }
 
+    /**
+     * Clears (if initialized) {@code tokenAliasMap} at the end of iterating through all balance
+     * changes in cryptoTransfer transaction.
+     */
+    public void clearTokenAliasMap() {
+        if (tokenAliasMap != null) {
+            tokenAliasMap.clear();
+        }
+    }
+
+    public Map<ByteString, Set<Id>> getTokenAliasMap() {
+        return tokenAliasMap;
+    }
+
+    private boolean shouldConstructTokenAliasMap(final BalanceChange change) {
+        return change.isForToken() && (tokenAliasMap == null || tokenAliasMap.isEmpty());
+    }
+
+    private void replaceAliasAndSetBalanceOnChange(
+            final BalanceChange change, final AccountID newAccountId) {
+        if (change.isForHbar()) {
```

Comment:
I don't think we need this any more since we don't deduct the fee from the change.